backend/routes/products.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+const {
+  getAllProducts,
+  getProduct,
+  getReviewsForProduct,
+  createNewProduct,
+  createNewReviewForProduct,
+  deleteProduct,
+  createRandomData,
+  getAllCategories,
+  deleteAllProducts,
+} = require('../controllers/productsController');
+
+const router = express.Router();
+
+// GET random data
+router.get('/get-random-data', createRandomData);
+
+// GET All Products
+router.get('/', getAllProducts);
```

Comment:
Elegant to structure your backend this way. Keeping the controllers separate is much more like something you'd see in a professional environment. Well done!